### ISSUE TITLE
Teplota

### ISSUE THREAD

**Issue by dobrakmato** [pexelnet/starving#18]
Hráč má určitú teplotu tela. Táto teplota je iba približne jedným z nasledujúcich stupňov teploty:
- podchladenie (32 C)
- nízka teplota (35 C)
- normálna teplota (37 C)
- vysoká teplota (39 C)
- prehriatie (42 C)

Hráč má zvýšenú teplotu počas choroby.
Hráč môže nadobudnúť nízku teplotu po dlhom pobyte v chladnom prostredí - podchladí sa. Po dlhodobom podchladení môže umrieť (postupná smrť #11 ).
### Prostredia:
- normálne
- studené
- teplé

Ak sa hráč nachádza v _normálnom_ prostredí s oblečením, jeho teplota je **stabilná**.
Ak sa hráč nachádza v _teplom_ alebo _normálnom_ prostredí bez oblečenia, jeho teplota je **stabilná**.
Ak sa hráč nachádza v _teplom_ prostredí s oblečením, jeho teplota **stúpa. Maximálne však do 39 stupňov.**
Ak sa hráč nachádza v _studenom_ prostredí s oblečením, jeho teplota je **stabilná**.
Ak sa hráč nachádza v _studenom_ prostredí bez oblečenia, jeho teplota **klesá** až na hranicu **32 stupňov, pri ktorej hráč umiera podchladením.**
Ak má hráč nízku alebo vysokú teplotu má halucinácie.

---
#33

**Comment by janscz** [comment]
Při dlouhodobém podchlazení bych dodal, že hrozí smrt a veškeré smysly má zpomalené (tzn. bude mít nějaký slow)

**Comment by kubob** [comment]
Spomalenie určite áno. Halucinácie... No neviem, povedzme, že áno.
Ako sa bude môcť zohriať? Čaj? Teplá sprcha? Ohníček?

**Comment by pitkes22** [comment]
Ohniček , pobit na slnku , teple jedlo by mohlo tiež fungovať ....

**Comment by dobrakmato** [comment]
Pamatam si ze sme sa zhodli na tom, ze ohnicek hracov nebude zohrievat. A pochybujem ze niekde zozenie teple jedlo :smile:.

@kubob http://en.wikipedia.org/wiki/Human_body_temperature#Cold

**Comment by pitkes22** [comment]
Zhodli sme sa nie na tom že ho nebude prehrievať ale nie na tom že ho nebude zohrievať ...

**Comment by dobrakmato** [comment]
Cize ohrievat ho mozu veci okrem choroby iba do 37 C?

**Comment by pitkes22** [comment]
Skvor by som spomenul že sme sa zhodli na tom že teplota nebude presne na stupne ale na Velmi nizku => Nizku => Normalnu => Vysoku čiže oheň bude zvišovať teplotu len z Leľmi nizkej a nizkej na normalnu ....

**Comment by dobrakmato** [comment]
Ano, ale ja aj ta preferujem menej cisel ako tie "priblizne popisne nazvy" :smile:

**Comment by janscz** [comment]
tak jak je to teď je sepsané si myslím že jo to Ok, jen bych dodal že pokud má hráč nemoc (a zvýšenou teplotu) tak si ji může snížit pobytem ve vodě nebo v chladném prostřední, dále. že se může z podchlazení ohřát ohněm nebo pobytem v teplém prostředí

Jinak Souhlasím

**Comment by dobrakmato** [comment]
#33

**Comment by dobrakmato** [comment]
Implemmentovana zmena teploty podla prostredia. https://github.com/dobrakmato/mcRPG/commit/63b1403573cb3a44acd2b073f32fbded82cc7938
